docs/reference/modules/discovery/bootstrapping.asciidoc

```diff
@@ -30,12 +30,12 @@ node:
 
 When you start a master-eligible node, you can provide this setting on the
 command line or in the `elasticsearch.yml` file. After the cluster has formed,
-this setting is no longer required and is ignored. It need not be set on
-master-ineligible nodes, nor on master-eligible nodes that are started to join
-an existing cluster. Note that master-eligible nodes should use storage that
-persists across restarts. If they do not, and `cluster.initial_master_nodes` is
-set, and a full cluster restart occurs, then another brand-new cluster will
-form and this may result in data loss.
+this setting is no longer required and is ignored. It should not be set on
+master-ineligible nodes, nor on master-eligible nodes that will be joining an
+existing cluster, nor when restarting a cluster. Note that master-eligible nodes
+should use storage that persists across restarts. If they do not, and
+`cluster.initial_master_nodes` is set, and a full cluster restart occurs, then
```

Comment:
```suggestion
`cluster.initial_master_nodes` is set during  a full cluster restart,
```

Reply:
I decided we can ditch this sentence. Other things can go wrong if not using persistent storage, and this point is covered elsewhere (on the dreaded Modules/Node page) anyway.